### PR TITLE
Feat/14 get variable day length

### PR DESCRIPTION
**Pull request by mattTara**
Variable day length with console printout

### CODE REVIEW

File: AstroCal/view/console_output.py [hunk 1]
```diff
@@ -45,6 +47,25 @@ def sun_menu():
     elif option == 2:
         print("Solar Eclipse:", control.getWhenSolEclipseLoc())
     elif option == 3:
+        amountOfDays = int(input('Enter amount of days: '))
+        now = datetime.now()
+        month = now.month
+        year = now.year
+        day = now.day
+        currentDay = datetime(year, month, day)
+        dayLenghts = control.getVariableDayLength(year, month, day, amountOfDays)
```
Comment (amyvezeau) [inline AstroCal/view/console_output.py]: Length is spelt wrong.

File: AstroCal/control/control.py [hunk 2]
```diff
@@ -206,3 +207,32 @@ def getMoonStatus():
...
+        daySetTime = datetime(daySunSet[0], daySunSet[1], daySunSet[2], daySunSet[3], daySunSet[4], int(daySunSet[5]))
+        
+        #get difference between sunrise and and sunset 
+        timeDiff = abs(daySetTime-dayRiseTime)
+        #convert time difference into seconds
+        seconds = timeDiff.seconds
+        #extract total hours from seconds
+        dayLightHours = (seconds / 60) / 60
+        #extract remaining minutes
+        dayLightMinutes = dayLightHours % 1
```
Comment (amyvezeau) [inline AstroCal/control/control.py]: This variable is called minutes which implies the value is the number of minutes, but it's the decimal value of hours. Might be confusing to future users of the code base. Could we use a variable name like dayLightHoursRemainder?

Comment (amyvezeau) [reply AstroCal/control/control.py]: or it looks like there could be a more python-ic (is that a word?) way https://stackoverflow.com/a/775075/5708566

**Review by amyvezeau — APPROVED** [review]

**Review by nmiles01 — APPROVED** [review]

File: tests/test_VariableDayLength [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import unittest
```
Comment (MellowYeller) [inline tests/test_VariableDayLength]: Nitpick here: this file is missing the `.py` extension. Just adding that to the file name would bring it in line with other files in the folder.

**Review by MellowYeller — CHANGES_REQUESTED** [review]

**Review by MellowYeller — APPROVED** [review]

**Pull request merged**